src/app/core/facades/app.facade.ts

```diff
@@ -90,6 +87,10 @@ export class AppFacade {
     return isAppTypeREST && !isBusinessCustomer ? 'privatecustomers' : 'customers';
   }
 
+  serverSetting$<T>(path: string) {
```

Comment:
for me it is not clear, what the path means, which is the starting point of the path